ipaplatform/fedora/paths.py

```diff
@@ -36,6 +37,9 @@ class FedoraPathNamespace(RedHatPathNamespace):
     NAMED_CRYPTO_POLICY_FILE = "/etc/crypto-policies/back-ends/bind.config"
     if HAS_NFS_CONF:
         SYSCONFIG_NFS = '/etc/nfs.conf'
+    if HAS_BIND_OPENSSL_ENGINE:
+        DNSSEC_OPENSSL_CONF = "/etc/ipa/dnssec/openssl.cnf"
+        DNSSEC_KEYFROMLABEL = "/usr/sbin/dnssec-keyfromlabel"
```

Comment:
``DNSSEC_KEYFROMLABEL`` is only defined in Fedora namespace and not used anywhere.

Reply:
```DNSSEC_KEYFROMLABEL = "/usr/sbin/dnssec-keyfromlabel-pkcs11" ``` in BasePathNamespace